pages/dkp/konvoy/2.1/release-notes/2.1.5/index.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: layout.pug
+navigationTitle: Konvoy 2.1.5 Release Notes
+title: Konvoy 2.1.5 Release Notes
+menuWeight: 50
+excerpt: View release-specific information for Konvoy 2.1.5
+enterprise: false
+beta: false
+---
+
+**D2iQ&reg; Konvoy&reg; version 2.1.5 was released on December 1st, 2022.**
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Konvoy[/button]
+
+To get started with Konvoy, [download](../../download/) and [install](../../choose-infrastructure/) the latest version of Konvoy.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Konvoy.
+
+## Fixes and Improvements
+
+### Wrong MachineDeployment (nodepool) names after Konvoy v1.8.5 to DKP v2.1.3 upgrade (D2IQ-93614)
+
+Corrected an issue where a `MachineDeployment` would be incorrectly named when running `dkp describe cluster` after upgrading from Konvoy v1.8. to v2.1.
+
+### Preprovisioned cluster adoption is broken (D2IQ-94144)
+
+Corrected an issue that prevented the adoption of pre-provisioned clusters in Konvoy 2.1.
```

Comment:
@alejandroEsc  - Do we need to update some docs in this release somewhere to mention KIB 1.5.2?  As I believe this release requires that version, right?   

If the above is true, then:

@mmcelroy91  - I think we should also update your KIB table when this releases to mention that DKP 2.1.5 requires KIB 1.5.2

Reply:
yes, that is correct. The wording should say "prevented the upgrade"